.travis.yml

```diff
@@ -16,7 +16,7 @@ env:
 
 cache:
   directories:
-    - vendor
+  - $GOPATH/pkg/mod
```

Comment:
Also, the missing cache directory breaks CI. I fixed it.
cf. https://go.dev/ref/mod#module-cache